### ISSUE TITLE
KeyValueProperty doesn't get properly converted to the babel equivelant

### ISSUE THREAD

**Issue by Markos-Th09** [coderaiser/swc-to-babel#2]
# Reproduction
Code:
```js
Object.defineProperty(exports, "__esModule", {
  value: true
});
```

<details>
<summary>swc ast:</summary>

```json
{
  "type": "Module",
  "span": {
    "start": 0,
    "end": 64,
    "ctxt": 0
  },
  "body": [
    {
      "type": "ExpressionStatement",
      "span": {
        "start": 0,
        "end": 64,
        "ctxt": 0
      },
      "expression": {
        "type": "CallExpression",
        "span": {
          "start": 0,
          "end": 63,
          "ctxt": 0
        },
        "callee": {
          "type": "MemberExpression",
          "span": {
            "start": 0,
            "end": 21,
            "ctxt": 0
          },
          "object": {
            "type": "Identifier",
            "span": {
              "start": 0,
              "end": 6,
              "ctxt": 0
            },
            "value": "Object",
            "optional": false
          },
          "property": {
            "type": "Identifier",
            "span": {
              "start": 7,
              "end": 21,
              "ctxt": 0
            },
            "value": "defineProperty",
            "optional": false
          }
        },
        "arguments": [
          {
            "spread": null,
            "expression": {
              "type": "Identifier",
              "span": {
                "start": 22,
                "end": 29,
                "ctxt": 0
              },
              "value": "exports",
              "optional": false
            }
          },
          {
            "spread": null,
            "expression": {
              "type": "StringLiteral",
              "span": {
                "start": 31,
                "end": 43,
                "ctxt": 0
              },
              "value": "__esModule",
              "raw": "\"__esModule\""
            }
          },
          {
            "spread": null,
            "expression": {
              "type": "ObjectExpression",
              "span": {
                "start": 45,
                "end": 62,
                "ctxt": 0
              },
              "properties": [
                {
                  "type": "KeyValueProperty",
                  "key": {
                    "type": "Identifier",
                    "span": {
                      "start": 49,
                      "end": 54,
                      "ctxt": 0
                    },
                    "value": "value",
                    "optional": false
                  },
                  "value": {
                    "type": "BooleanLiteral",
                    "span": {
                      "start": 56,
                      "end": 60,
                      "ctxt": 0
                    },
                    "value": true
                  }
                }
              ]
            }
          }
        ],
        "typeArguments": null
      }
    }
  ],
  "interpreter": null
}
```

</details>

<details>
<summary>Output babel AST</summary>

```json
{
    "type": "File",
    "program": {
        "type": "Program",
        "body": [
            {
                "type": "ExpressionStatement",
                "expression": {
                    "type": "CallExpression",
                    "callee": {
                        "type": "MemberExpression",
                        "object": {
                            "type": "Identifier",
                            "name": "Object"
                        },
                        "property": {
                            "type": "Identifier",
                            "name": "defineProperty"
                        },
                        "computed": false
                    },
                    "arguments": [
                        {
                            "type": "Identifier",
                            "name": "exports"
                        },
                        {
                            "type": "StringLiteral",
                            "value": "__esModule",
                            "raw": "\"__esModule\"",
                            "extra": {
                                "raw": "\"__esModule\""
                            }
                        },
                        {
                            "type": "ObjectExpression",
                            "properties": [
                                {
                                    "type": "KeyValueProperty",
                                    "key": {
                                        "type": "Identifier",
                                        "span": {
                                            "start": 49,
                                            "end": 54,
                                            "ctxt": 0
                                        },
                                        "value": "value",
                                        "optional": false
                                    },
                                    "value": {
                                        "type": "BooleanLiteral",
                                        "span": {
                                            "start": 56,
                                            "end": 60,
                                            "ctxt": 0
                                        },
                                        "value": true
                                    }
                                }
                            ]
                        }
                    ]
                }
            }
        ],
        "interpreter": null,
        "directives": [],
        "sourceType": "module"
    },
    "comments": []
}
```

</details>

<details>
<summary>Expected AST</summary>

```json
{
  "type": "File",
  "start": 0,
  "end": 65,
  "loc": {
    "start": {
      "line": 1,
      "column": 0,
      "index": 0
    },
    "end": {
      "line": 4,
      "column": 0,
      "index": 65
    }
  },
  "errors": [],
  "program": {
    "type": "Program",
    "start": 0,
    "end": 65,
    "loc": {
      "start": {
        "line": 1,
        "column": 0,
        "index": 0
      },
      "end": {
        "line": 4,
        "column": 0,
        "index": 65
      }
    },
    "sourceType": "module",
    "interpreter": null,
    "body": [
      {
        "type": "ExpressionStatement",
        "start": 0,
        "end": 64,
        "loc": {
          "start": {
            "line": 1,
            "column": 0,
            "index": 0
          },
          "end": {
            "line": 3,
            "column": 3,
            "index": 64
          }
        },
        "expression": {
          "type": "CallExpression",
          "start": 0,
          "end": 63,
          "loc": {
            "start": {
              "line": 1,
              "column": 0,
              "index": 0
            },
            "end": {
              "line": 3,
              "column": 2,
              "index": 63
            }
          },
          "callee": {
            "type": "MemberExpression",
            "start": 0,
            "end": 21,
            "loc": {
              "start": {
                "line": 1,
                "column": 0,
                "index": 0
              },
              "end": {
                "line": 1,
                "column": 21,
                "index": 21
              }
            },
            "object": {
              "type": "Identifier",
              "start": 0,
              "end": 6,
              "loc": {
                "start": {
                  "line": 1,
                  "column": 0,
                  "index": 0
                },
                "end": {
                  "line": 1,
                  "column": 6,
                  "index": 6
                },
                "identifierName": "Object"
              },
              "name": "Object"
            },
            "computed": false,
            "property": {
              "type": "Identifier",
              "start": 7,
              "end": 21,
              "loc": {
                "start": {
                  "line": 1,
                  "column": 7,
                  "index": 7
                },
                "end": {
                  "line": 1,
                  "column": 21,
                  "index": 21
                },
                "identifierName": "defineProperty"
              },
              "name": "defineProperty"
            }
          },
          "arguments": [
            {
              "type": "Identifier",
              "start": 22,
              "end": 29,
              "loc": {
                "start": {
                  "line": 1,
                  "column": 22,
                  "index": 22
                },
                "end": {
                  "line": 1,
                  "column": 29,
                  "index": 29
                },
                "identifierName": "exports"
              },
              "name": "exports"
            },
            {
              "type": "StringLiteral",
              "start": 31,
              "end": 43,
              "loc": {
                "start": {
                  "line": 1,
                  "column": 31,
                  "index": 31
                },
                "end": {
                  "line": 1,
                  "column": 43,
                  "index": 43
                }
              },
              "extra": {
                "rawValue": "__esModule",
                "raw": "\"__esModule\""
              },
              "value": "__esModule"
            },
            {
              "type": "ObjectExpression",
              "start": 45,
              "end": 62,
              "loc": {
                "start": {
                  "line": 1,
                  "column": 45,
                  "index": 45
                },
                "end": {
                  "line": 3,
                  "column": 1,
                  "index": 62
                }
              },
              "properties": [
                {
                  "type": "ObjectProperty",
                  "start": 49,
                  "end": 60,
                  "loc": {
                    "start": {
                      "line": 2,
                      "column": 2,
                      "index": 49
                    },
                    "end": {
                      "line": 2,
                      "column": 13,
                      "index": 60
                    }
                  },
                  "method": false,
                  "key": {
                    "type": "Identifier",
                    "start": 49,
                    "end": 54,
                    "loc": {
                      "start": {
                        "line": 2,
                        "column": 2,
                        "index": 49
                      },
                      "end": {
                        "line": 2,
                        "column": 7,
                        "index": 54
                      },
                      "identifierName": "value"
                    },
                    "name": "value"
                  },
                  "computed": false,
                  "shorthand": false,
                  "value": {
                    "type": "BooleanLiteral",
                    "start": 56,
                    "end": 60,
                    "loc": {
                      "start": {
                        "line": 2,
                        "column": 9,
                        "index": 56
                      },
                      "end": {
                        "line": 2,
                        "column": 13,
                        "index": 60
                      }
                    },
                    "value": true
                  }
                }
              ]
            }
          ]
        }
      }
    ],
    "directives": []
  },
  "comments": []
}
```

</details>

# Expected behaviour
KeyValueProperty nodes should be replaced with the proper ObjectProperty node equivelant

# Current behaviour
KeyValueProperty nodes aren't replaced at all producing an invalid ast

**Comment by coderaiser** [comment]
This is a very good idea! We definitely need support of `KeyValueProperty`.

**Comment by coderaiser** [comment]
Landed in [v1.22.0](https://github.com/coderaiser/swc-to-babel/releases/tag/v1.22.0) 🎉 . Is it works for you?

**Comment by Markos-Th09** [comment]
> Landed in [v1.22.0](https://github.com/coderaiser/swc-to-babel/releases/tag/v1.22.0) 🎉 . Is it works for you?

Yes it is. Although I found more issues which I am going to push a pr fixing today.
